.github/workflows/ci.yaml

```
@@ -29,7 +29,10 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        node-version: ['lts/gallium', 'lts/hydrogen', 'lts/iron', 'lts/*']
+        node-version:
+         - 'lts/gallium'
```

yamllint (1.29.0-1)

[warning] wrong indentation: expected 10 but found 9

(indentation)